BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -441,3 +474,21 @@ extension BlueprintView {
         }
     }
 }
+
+extension BlueprintView.NativeViewController {
+    
+    /// A context value passed to the ``BlueprintView/NativeViewController`` instance as the view tree is updated by blueprint.
+    /// Contains information relevant to the correct construction and management of the view tree.
+    struct UpdateContext {
```

Comment:
This isn't a needed change; but I did it since it shortens up the callsites and makes it easier to add additional information here without appending tons of parameters (like animation state, which I was planning on looking at reviving soon).